### PR TITLE
chore(docs): fix broken links to renku-python

### PR DESCRIPTION
**Pull request by bethcg**
This pull request collects several commits to solve some broken links to renku-python in the documentation.

### CODE REVIEW

**Review by bethcg — COMMENTED** [review]
Solved broken links in the documentation for renku-python

**Comment by rokroskar** [comment]
@bethcg what's the status here, could we merge it soon?

File: docs/reference/templates.rst [hunk 1]
```diff
@@ -430,7 +430,7 @@ template repository as an argument to the ``renku init`` command. If you do
 this, we recommend that you **also** specify a tag (or a commit) when creating
 a new project from a custom repository to ensure that the action is reproducible.
 You can find further details in
-`renku init docs <https://renku.readthedocs.io/en/latest/renku-python/docs/reference/commands.html#use-a-different-template>`_.
+`renku init docs <https://renku-python.readthedocs.io/en/stable/reference/commands/template.html#set-a-template>`_.
```
[the remark this answers is not in the document]
Comment (bethcg) [reply docs/reference/templates.rst]: Completed

File: docs/topic-guides/sessions/customizing-sessions.rst [hunk 2]
```diff
@@ -115,4 +115,4 @@ Getting Help
 If you are stuck with a specific modification you'd like to make, do reach out to the
 `Renku community forum <https://renku.discourse.group>`_!
 
-.. _`renku config command`: https://renku.readthedocs.io/en/latest/renku-python/docs/reference/commands.html#module-renku.ui.cli.config
+.. _`renku config command`: https://renku-python.readthedocs.io/en/stable/reference/commands/config.html
```
Comment (rokroskar) [inline docs/topic-guides/sessions/customizing-sessions.rst:118]: can't this also use the internal ref?

**Review by rokroskar — CHANGES_REQUESTED** [review]

**Comment by rokroskar** [comment]
@bethcg I updated this PR and rebased it on the current release branch. We still ahve these broken links in the wild so we should fix them :)

**Review by rokroskar — APPROVED** [review]

**Review by Panaetius — APPROVED** [review]

**Pull request merged**